packages/next-plugin-sentry/src/on-init-server.js

```diff
@@ -1,11 +1,39 @@
-import * as Sentry from '@sentry/node'
+import { init } from '@sentry/node'
+import { RewriteFrames } from '@sentry/integrations'
+import getConfig from 'next/config'
+
+import { getDsn, getRelease } from '../env'
+import { serverConfig } from '../config'
 
 export default async function initServer() {
-  // by default `@sentry/node` is configured with defaultIntegrations
-  // which capture uncaughtExceptions and unhandledRejections
-  // see here for more https://github.com/getsentry/sentry-javascript/blob/46a02209bafcbc1603c769476ba0a1eaa450759d/packages/node/src/sdk.ts#L22
-  Sentry.init({
-    dsn: process.env.SENTRY_DSN,
-    release: process.env.SENTRY_RELEASE,
+  /**
+   * TODO(kamil): Unify SDK configuration options.
+   * RuntimeConfig cannot be used for callbacks and integrations as it supports only serializable data.
+   **/
+  const { serverRuntimeConfig = {}, publicRuntimeConfig = {} } =
+    getConfig() || {}
+  const runtimeConfig =
+    serverRuntimeConfig.sentry || publicRuntimeConfig.sentry || {}
```

Comment:
`next/config` should not be used. This is not compatible with Next.js' SSG mode and only exists for legacy reasons.